rocks/RocksTransaction.java

```diff
@@ -120,6 +120,10 @@ public void close() {
         }
     }
 
+    public Reasoner reasoner() {
```

Comment:
We need to be able to access the reasoner in order to make assertions in tests about it's behaviour.

Reply:
did this turn out to have to be public?